components/footer.js

```diff
@@ -1,6 +1,7 @@
+import React from 'react'
 import Link from 'next/link'
 
-export default () => (
+const Footer = () => (
```

Comment:
Moving to a `const Component` style was a bit of a refactor to support displayName and propTypes. It seemed less destructive then converting everything to React.Component and jamming those things in the constructor.